docs/source/github.rst

```diff
@@ -156,6 +156,24 @@ Check compatibility and propose a Pull Request. You should see your pushes on th
 
 You're ready to collaborate to any Open-Source repository on GitHub!
 
+modify and import the deepxde library
+***************************************************************
+To modify and test future changes to the deepxde library, it is recommended to follow these steps:
+
+clone the deepxde library repository
+
+Once the repository is cloned on the server, use the following command to define
+the address where it is located.
```

Comment:
use the appropriate format to include code blocks (for example, see https://github.com/pescap/EasyHPC/blob/c0d68039398d2798c3346961aa10677472de918c/docs/source/cicd.rst)